tests/integration/test_logcollector/test_macos/test_macos_file_status_basic.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import re
+import pytest
+import wazuh_testing.logcollector as logcollector
+
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools import LOGCOLLECTOR_FILE_STATUS_PATH
+from wazuh_testing.remote import check_agent_received_message
+from wazuh_testing.tools.monitoring import wait_file
+from wazuh_testing.tools.file import read_json
+from time import sleep
+
+# Marks
+pytestmark = [pytest.mark.darwin, pytest.mark.tier(level=0)]
+
+# Configuration
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_macos_file_status_basic.yaml')
+
+parameters = [{'ONLY_FUTURE_EVENTS': 'yes'}, {'ONLY_FUTURE_EVENTS': 'no'}]
+metadata = [{'only-future-events': 'yes'}, {'only-future-events': 'no'}]
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+configuration_ids = [f'{x["ONLY_FUTURE_EVENTS"]}' for x in parameters]
+
+# Time in seconds to update the file_status.json
+file_status_update_time = 4
+
+local_internal_options = {'logcollector.vcheck_files': str(file_status_update_time)}
+
+
+@pytest.fixture(scope='module')
+def get_local_internal_options():
+    """Get configurations from the module."""
+    return local_internal_options
+
+
+# Fixtures
+@pytest.fixture(scope='module', params=configurations, ids=configuration_ids)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture(scope='module')
+def get_connection_configuration():
+    """Get configurations from the module."""
+    return logcollector.DEFAULT_AUTHD_REMOTED_SIMULATOR_CONFIGURATION
+
+
+def extra_configuration_before_yield():
+    """Delete file status file."""
+    os.remove(LOGCOLLECTOR_FILE_STATUS_PATH) if os.path.exists(LOGCOLLECTOR_FILE_STATUS_PATH) else None
+
+
+def test_macos_file_status_basic(get_local_internal_options, configure_local_internal_options, get_configuration,
+                                 configure_environment, get_connection_configuration, init_authd_remote_simulator,
+                                 restart_logcollector):
```

Comment:
After talking with @snaow we have considered that new development or changes into existing tests should follow new repository standards about Wazuh daemons. For now, it is required to use fixtures to start only required daemons, and turn them off when the test is over. That means, that the default status of Wazuh would be off and each test should be responsible for daemons handling.
In case of the need of all Wazuh daemon, the test should disable every module not necessary for the testing in ossec.conf.

The reasons for this change are:

- Faster testing
- More encapsulated tests

Also, the use of simulators is not allowed.